### DISCUSSION TITLE
pamac-cli can be installed without issue

### DISCUSSION THREAD

**Discussion by MWatter** [sileshn/ManjaroWSL2#16] [Ideas]
I know there's been some issues with pamac in the past but I did a ManjaroWSL2 install today and had no issues installing `pamac-cli` with `sudo pacman -Syu  pamac-cli`. 

I've now used pamac in my terminal for several package installs and it properly requests credentials at the command line.

**Comment by sileshn** [comment]
pamac-cli is already included in builds after June 2023. There is no need to install it.
